.github/workflows/swift-toolchain.yml

```diff
@@ -21,16 +22,40 @@ on:
         description: 'Emit PDBs (Debug Info)'
         default: false
         type: boolean
+
       signed:
         description: 'Code Sign'
         default: false
         type: boolean
+
+      default_runner:
```

Comment:
I'm not convinced about the changes for the runners being an input. That's confusing for most people - it requires a good understanding of the labels which is not necessarily visible to everyone.

Reply:
We can remove this for now then, and add it back once we think this workflow should expose an API for choosing the runners more generally.